.github/workflows/ci_workflows.yml

```diff
@@ -39,3 +39,37 @@ jobs:
       run: make check
     - name: make distcheck
       run: make distcheck
+
+  tests-meson:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: linux (with gcc and meson)
+            os: ubuntu-latest
+
+          - name: MacOS (with clang and meson)
```

Comment:
Shall we add windows too? Would be nice to show we can build on it!

Reply:
I'm not overly familiar with Windows, but I took a stab at this.

My workflow test results are here: https://github.com/eli-schwartz/erfa/actions/runs/3148176374/jobs/5118436048

```
[255/255] Linking target src/t_erfa_c_extra.exe
FAILED: src/t_erfa_c_extra.exe src/t_erfa_c_extra.pdb 
"link"  /MACHINE:x64 /OUT:src/t_erfa_c_extra.exe src/t_erfa_c_extra.exe.p/t_erfa_c_extra.c.obj "/nologo" "/release" "/nologo" "/DEBUG" "/PDB:src\t_erfa_c_extra.pdb" "src\erfa.lib" "/SUBSYSTEM:CONSOLE" "kernel32.lib" "user32.lib" "gdi32.lib" "winspool.lib" "shell32.lib" "ole32.lib" "oleaut32.lib" "uuid.lib" "comdlg32.lib" "advapi32.lib"
LINK : fatal error LNK1181: cannot open input file 'src\erfa.lib'
```

I think it's saying that because you don't have https://learn.microsoft.com/en-us/cpp/build/reference/module-definition-dot-def-files?view=msvc-170 it was unable to build a shared DLL that contains public symbols (MSVC hides all symbols by default IIRC, unlike unix compilers that expose them all by default), although configuring meson with `-Ddefault_library=static` would produce a static-only library that would likely work.

Reply:
Okay, pushed that with hardcoding the static library for now on Windows.